spec/requests/events/comments/create_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
```

Comment:
Unfortunately, this test suite will only verify the API responses. It lacks some tests verifying that the comment has been created with the appropriate associations/attributes.